src/jade/config/run_config.py

```
@@ -157,39 +160,47 @@ class EnvironmentVariables:
+            raise ConfigError(
+                "Scheduler command is needed if run_mode is 'job', please provide one"
+            )
+        code_job_template = {}
+        if self.code_job_template is not None and self.exe_cfg_root is not None:
+            for code, path in self.code_job_template.items():
+                if not os.path.isabs(path) and self.exe_cfg_root is not None:
+                    path = os.path.join(self.exe_cfg_root, path)
+                if not os.path.isfile(path):
+                    raise ConfigError(
+                        f"Job script template {path} not found, please check and re-run"
+                    )
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)